docs/topics/whatsnew-eap.md

```diff
@@ -80,6 +80,49 @@ These features are available in preview, and we encourage you to try them and sh
 
 [See the full list of Kotlin language design features and proposals](kotlin-language-features-and-proposals.md).
 
+### Non-local break and continue
+
+Kotlin %kotlinEapVersion% adds a preview of another long-awaited feature, an opportunity to use non-local `break` and
```

Comment:
Isn't it too marketish even for a blog post?

Kotlin %kotlinEapVersion% introduces a preview of a new feature: support for non-local` break` and `continue`.

Reply:
I don't think so, and I don't usually use such phrasing, but here it's justified, the feature request is 12 years old.